src/main/blackbox/blackbox.h

```diff
@@ -20,10 +20,10 @@
 
 #pragma once
 
-#include "platform.h"
 #include "build/build_config.h"
 #include "common/time.h"
 #include "pg/pg.h"
+#include "platform.h"
```

Comment:
This will break a lot of things, since target dependent defines are included through `platform.h`. In general, there is a number of cases where include order matters, and has to be non - alphabetical. In this respect, force-reordering here is wrong.

Reply:
As I asked you before, can you please add a document listing this, and all the other rules that you want to enforce, in a human readable form (you can use `docs/development/CodingStyle.md` as a template)?

Without this I fear this will just lead to contributors applying machine formatting blindly, and relying on whoever reviews the code to spot and fix things that have been broken by the machine formatting.